server/controller/src/main/resources/application.yaml

```diff
@@ -158,6 +158,9 @@ mybatis:
   mapper-locations: classpath*:mapper/**/*.xml
 
 server:
+  tomcat:
+    threads:
+      max: ${SW_CONTROLLER_MAX_THREADS:500}
```

Comment:
Does this number include the built-in thread pool?

Reply:
yes, renamed it